applications/ecallmgr/src/ecallmgr_fs_route.erl

```diff
@@ -336,8 +336,9 @@ reply_forbidden(Section, Node, FetchId) ->
     end.
 
 -spec reply_affirmative(atom(), atom(), ne_binary(), ne_binary(), wh_json:object()) -> 'ok'.
-reply_affirmative(Section, Node, FetchId, CallId, JObj) ->
+reply_affirmative(Section, Node, FetchId, CallId, _JObj) ->
     lager:info("received affirmative route response for request ~s", [FetchId]),
+    JObj = wh_json:set_value(<<"Fetch-Section">>, wh_util:to_binary(Section), _JObj),
```

Comment:
@lazedo not really proper to actually use _JObj, we make exceptions for logging only..... PreFetchJObj?